### PR TITLE
RC_Channel: correct description of standby option

### PR DESCRIPTION
**Pull request by peterbarker**
Do not merge before https://github.com/ArduPilot/ardupilot/pull/12595/files

### CODE REVIEW

File: libraries/RC_Channel/RC_Channel.h [hunk 1]
```diff
@@ -173,7 +173,7 @@ class RC_Channel {
         DRIFT     =           73, // drift mode
         SAILBOAT_MOTOR_3POS = 74, // Sailboat motoring 3pos
         SURFACE_TRACKING =    75, // Surface tracking upwards or downwards
-        STAND_BY  =           76, // Standy signal magnitude
+        STAND_BY  =           76, // Standy mode
```
Comment (amilcarlucas) [inline libraries/RC_Channel/RC_Channel.h]: Shouldn't this be:
` STANDBY  =           76, // Standy mode`

Comment (rmackay9) [reply libraries/RC_Channel/RC_Channel.h]: And why is it "Standy" in the comments? :-)

Comment (peterbarker) [reply libraries/RC_Channel/RC_Channel.h]: Fixed the comment.

Canonicalised on `STANDBY` rather than `STAND_BY`.

**Comment by rmackay9** [comment]
Great, thanks.  I think standby is better than stand_by.  Hopefully @lthall agrees..

**Pull request merged**